src/backends/torch/torchlib.cc

```diff
@@ -1117,7 +1149,88 @@ namespace dd
             if (_module._native != nullptr)
               output = _module._native->cleanup_output(output);
 
-            if (_classification)
+            if (bbox)
+              {
+                // Supporting only Faster RCNN format at the moment.
```

Comment:
Since RetinaNet is mentioned in the PR, this only applies to Faster-RCNN, correct ?

Reply:
I forgot to change the comment, this works for RetinaNet as well.
There are two pure detection models in torchvision: Faster RCNN and RetinaNet, and they have the same output format.